### PR TITLE
create Android project with empty activity

### PR DESCRIPTION
**Pull request by eszterjudit**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext.kotlin_version = '1.2.61'
```
Comment (nyekilajos) [inline build.gradle]: It seems there is a newer kotlin plugin already: 1.2.70, please use that.

File: app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation"org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'com.android.support:appcompat-v7:27.1.1'
```
Comment (nyekilajos) [inline app/build.gradle]: Please use jetpack dependencies with the latest versions (as we use 3.2 gradle plugin which is not on stable yet, we can use here the not stable versions as well):

```
implementation "androidx.appcompat:appcompat:1.0.0-rc02"
implementation "androidx.constraintlayout:constraintlayout:2.0.0-alpha2"

androidTestImplementation "androidx.test:runner:2.0.0-rc01"
androidTestImplementation "androidx.test.espresso:espresso-core:3.1.0-alpha4"
```

File: app/build.gradle [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+android {
+    compileSdkVersion 27
+    defaultConfig {
+        applicationId "com.example.weathercheckerapp"
+        minSdkVersion 21
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
```
Comment (nyekilajos) [inline app/build.gradle]: Use the jetpack version here as well:

```
testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner
```

File: app/build.gradle (hunk 2 above)
Comment (eszterjudit) [reply app/build.gradle]: added "androidx.test:runner:1.1.0-alpha4" instead of "2.0.0-rc01"

**Pull request merged**